tmux.c

```diff
@@ -127,6 +127,7 @@ make_label(const char *label, char **cause)
 		free(base);
 		goto fail;
 	}
+    free(base);
```

Comment:
This is a partial fix. Notice the other places that `goto fail;`. This is a situation where it is better to initialize the var to `NULL` where it is defined and then do `if (base) free(base);` in the error path.

Reply:
Yes that would work, although free(NULL) is always OK so no need for an if guard.

Reply:
Note the spaces here. Tab should be used.